appengine/standard/firebase/firetactoe/templates/fire_index.html

```diff
@@ -0,0 +1,48 @@
+<html>
+  <head>
+    {% include "_firebase_config.html" %}
+    <link rel="stylesheet" type="text/css" href="static/main.css" />
+    <script src="static/main.js"></script>
+    <script>
+      initGame('{{ game_key }}', '{{ me }}', '{{ token }}', '{{ channel_id }}',
```

Comment:
Why?
I like it - it puts the information precisely where it's needed, and the templating system makes sure it's safe.